examples/ciao/group_vars/all

```diff
@@ -1,41 +1,28 @@
 ---
 database_root_password: secret
 
-glance_fqdn: image.example.com
-glance_user_password: secret
-glance_database_password: secret
+keystone_fqdn: identity.example.com
+keystone_admin_password: secret
+keystone_database_password: secret
+keystone_root_domain: example.com
+keystone_p12password: secret
 
```

Comment:
yes, it is documented on os-keystone/README.md
https://github.com/clearlinux/clear-config-management/blob/doc/roles/os-keystone/README.md